package.json

```diff
@@ -84,7 +96,7 @@
     "start-bundle-server": "rm -rf debug && BROWSER=none node scripts/start.js",
     "start-dev-server": "npx ts-node-dev --project server/tsconfig.json server/index.ts",
     "build": "rm -rf build && node scripts/build.js",
-    "test": "node scripts/test.js",
+    "lint": "eslint src &&  tsc --project ./tsconfig.json --noEmit",
```

Comment:
Refer to auth-react or other js frontend libraries for all the scripts we use for checking if the code is linted and pretty (also make changes to the pre commit for this)

Reply:
pre-commit is updated